packages/eslint-plugin-mobx/rollup.config.js

```diff
@@ -13,6 +14,7 @@ export default [
         babelHelpers: "bundled",
         exclude: "**/node_modules/**",
       }),
+      json(),
```

Comment:
purpose?

Reply:
In general for this: https://github.com/mobxjs/mobx/pull/3942/commits/5c9f4bcd3d3b067fa3cb598b9b5e02be068f6625#diff-ea6b7dd8ae5b79523bfec5004a125b001785f118b2ff73dded233e51648b1e38R3 . Eslint recommends adding name and version into `meta` to improve caching: https://eslint.org/docs/latest/extend/plugins#meta-data-in-plugins 

A common approach in other plugins is simply `require`/`import` `package.json` already on the consumer machine (in runtime) - this way the meta version will always match the package version.

However, I did not take into account that in this case the compilation process occurs, and the package is copied to the `dist` as a whole (and I don't know how you synchronize the process of increasing the version and build, so it looks unreliable). It seems that it is better to use a slightly more complex approach from the eslint docs. So, I rewrote it.

There are also options with manual control of `meta` values (that shooting yourself in the foot sooner or later) or somehow integrate the process of filling this data into your build processes, but I am not familiar with them and am not sure that I can do it well.

Reply:
Thank you for elaborating. If I follow correctly, the current approach works and doesn't require extra dep.